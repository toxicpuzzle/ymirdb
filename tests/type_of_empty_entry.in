SET a 1 
POP a
TYPE a
bye