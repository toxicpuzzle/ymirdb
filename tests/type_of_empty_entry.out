> ok

> 1

> simple

> bye
